README.md

```diff
@@ -17,7 +19,16 @@ Sockeye requires:
 - numpy
 
 ## Installation
-You have two options for installing sockeye: pip and directly from source.
+
+### For AWS DeepLearning AMI users
+
+AWS DeepLearning AMI users need to run only one line in order to install the sockeye:
+
+```bash
+> sudo pip3 install sockeye --no-deps
+```
+
+For non DLAMI users - you have two options for installing sockeye: pip and directly from source.
```

Comment:
"For other environments, you can choose between installing via pip or directly from source."